src/modules/organization/createOrg/create-org-form/useCreateOrgForm.ts

```diff
@@ -0,0 +1,243 @@
+import { computed, Ref, ref, SetupContext, watch } from "vue";

```

Comment:
This component is very long. Can we separate it into another hook then import it back?

Reply:
What you mean by hook?